examples/huggingface/README.md

```diff
@@ -0,0 +1,103 @@
+# Implementation of NLP algorithms using Zenml
+
+NLP is a branch of machine learning that is about helping systems to understand natural text and spoken words in the same way that humans do.
+
+The following is a list of common NLP tasks:
+
+- Classification of sentences: sequence-classification
+- Classification of each words in a sentence: token-classification
+- Extraction of answer from a context text: question-answering
+- Text generation using prompt: text-generation
+- Translation: text-translation
+
+These examples demonstrate how we can use zenml and huggingface transformers to build, train, & test NLP models.
+
+## Sequence Classification
+
+Sequence Classification is an NLP/NLU task, where we assign labels to a given text, i.e. sentiment classification, natual langauge inference etc. In this example, we will train a sentiment classification model using the `imdb` dataset.
+
+- Load dataset: Load sequence-classification dataset in this case it is `imdb` dataset
+- Load pre-trained tokenizer: Load pre-trained tokenizer from huggingface transformers.
+- Tokenize and Prepare dataset for training: Use pre-trained tokenizer to tokenize and encode dataset into ids along with labels
+- Build and Train Model: You can build model or use pre-trained models from huggingface transformers. Use encoded dataset to train model.
+- Evaluate: Evaluate model loss and accuracy
+
+
+## Token Classification
+
+Token Classification is an NLP/NLU task, where we assign labels to tokens in a text, i.e. Name entity recognition, Part of speech tagging etc. In this example, we will train a NER model using the `conll2003` dataset.
+
+- Load dataset: Load token-classification dataset in this case it is `conll2003` dataset
+- Load pre-trained tokenizer: Load pre-trained tokenizer from huggingface transformers.
+- Tokenize and Prepare dataset for training: Use pre-trained tokenizer to tokenize and encode dataset into ids along with labels
+- Build and Train Model: You can build model or use pre-trained models from huggingface transformers. Use encoded dataset to train model.
+- Evaluate: Evaluate model loss and accuracy
+
```

Comment:
I would also think, that a tiny example of the task taken out of the dataset could make sense here